pims/bioformats.py

```diff
@@ -173,10 +173,11 @@ def fnw(fn1=fn, naame=name, paramcount=paramcount):
                     # function is not supported by this specific reader
                     pass
 
+        self.fields = list(filter(lambda x: x[:2] != '__', dir(self)))
```

Comment:
I think it would make more sense if this is put in the for loop above, at line 171: `self.fields.append(name[3:])`